generate.pl

```diff
@@ -179,12 +179,18 @@ sub die_with_sample {
         print $fh $patch;
       }
 
-      if (defined $release->{test_parallel} && $release->{test_parallel} eq "no") {
+      $release->{test_parallel} //= "yes";
```

Comment:
Sounds like `test_parallel` could do with a name change now since its not just checking if we want parallel tests but whether we want tests or not at all.

Reply:
I suppose so :sweat_smile:

What do you think about something like `tests: parallel` vs `tests: serial` vs `tests: no` ?

Reply:
Could also call it `run_tests: {parallel,serial,no}` :+1:

Reply:
Updated with `run_tests: {parallel,serial,no}` for now while we discuss EOLs / Actions. :+1: